server/src/test/java/org/apache/druid/client/CachingClusteredClientTest.java

```diff
@@ -2200,9 +2200,7 @@ public void run()
       serverExpectationList.add(serverExpectations);
       for (int j = 0; j < numChunks; ++j) {
         DruidServer lastServer = servers[random.nextInt(servers.length)];
-        if (!serverExpectations.containsKey(lastServer)) {
-          serverExpectations.put(lastServer, new ServerExpectations(lastServer, makeMock(mocks, QueryRunner.class)));
-        }
+        serverExpectations.putIfAbsent(lastServer, new ServerExpectations(lastServer, makeMock(mocks, QueryRunner.class)));
```

Comment:
Another good candidate for `computeIfAbsent`.

Reply:
Done.